### PR TITLE
tests for estCrossVal

### PR DESCRIPTION
**Pull request by lindsaycarr**
Characterizing output of estCrossVal through these tests.

### CODE REVIEW

File: tests/testthat/tests_estCrossVal.R [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  eList <- Choptank_eList
+  info_stale <- getInfo(eList)
+  daily_stale <- getDaily(eList)
+  sample_stale <- getSample(eList)
+  
+  info_orig <- info_stale[, 1:(which(names(info_stale) == "bottomLogQ") - 1)]
+  daily_orig <- daily_stale[, 1:(which(names(daily_stale) == "yHat") - 1)]
+  sample_orig <- sample_stale[, 1:(which(names(sample_stale) == "yHat") - 1)]
+  surfaces_orig <- NA
+  eList_orig <- mergeReport(info_orig, daily_orig, sample_orig, surfaces_orig)
```
Comment (aappling-usgs) [inline tests/testthat/tests_estCrossVal.R]: if there are going to be more than 3 of these code chunks (lines 7-16, this PR and previous couple), probably worth moving this code into a testing helper function. i think you can do this by naming a file 'helper-xxx.R' in the tests folder?

Comment (lindsaycarr) [inline tests/testthat/tests_estCrossVal.R]: oh right - good point. I forgot about that feature! Should I include that in this PR or do a separate one?

Comment (aappling-usgs) [inline tests/testthat/tests_estCrossVal.R]: separate is fine

File: tests/testthat/tests_estCrossVal.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  # execute cross validation
+  numDays <- length(daily_orig$DecYear)
+  DecLow <- daily_orig$DecYear[1]
+  DecHigh <- daily_orig$DecYear[numDays]
+  sample_crossval <- estCrossVal(numDays,DecLow,DecHigh,sample_orig)
+  
+  # estCrossVal adds three columns to Sample
+  expect_equal(ncol(sample_crossval) - ncol(sample_orig), 3)
+  expect_false(any(c("yHat", "SE", "ConcHat") %in% names(sample_orig)))
+  expect_true(all(c("yHat", "SE", "ConcHat") %in% names(sample_crossval)))
```
Comment (aappling-usgs) [inline tests/testthat/tests_estCrossVal.R]: here's where `setdiff` might actually simplify the code:
```
new_sample_cols <- setdiff(names(sample_crossval), names(sample_orig))
expect_equal(sort(new_sample_cols), sort(c("yHat", "SE", "ConcHat")))
```

Comment (lindsaycarr) [inline tests/testthat/tests_estCrossVal.R]: :+1:  ah yes - I like that! I'll update and then I can merge when builds pass.

**Comment by lindsaycarr** [comment]
This was part of #102

**Pull request merged**